src/mame/sgi/indigo.cpp

```diff
@@ -224,6 +224,7 @@ void indigo4k_state::indigo4k(machine_config &config)
 
 	SGI_MC(config, m_mem_ctrl, m_maincpu, m_eeprom);
 	m_mem_ctrl->eisa_present().set_constant(0);
+	m_mem_ctrl->set_clock(50000000);
```

Comment:
If the memory controller needs the clock set, you’re better off changing the constructor so it doesn’t default the clock frequency to zero.

Reply:
I was intended to do so but I'm unable to find a example in tree. Do you mind to point me one?

Reply:
Look at the first constructor here: https://github.com/mamedev/mame/blob/mame0253/src/mame/sgi/sgi.h#L21

See that it doesn’t take a clock parameter, and it passes zero as the clock frequency to the standard constructor.

Compare that to the constructors here: https://github.com/mamedev/mame/blob/mame0253/src/devices/machine/acorn_memc.h#L25

Notice that all constructors take a clock frequency.

Reply:
Thanks! That's helpful, not a C++ guru lol